pysbd/clean/rules.py

```diff
@@ -64,7 +64,8 @@ class CleanRules(object):
 
 class HTML(object):
     # Rubular: http://rubular.com/r/9d0OVOEJWj
-    HTMLTagRule = Rule(r"<\/?\w+((\s+\w+(\s*=\s*(?:\".*?\"|'.*?'|[\^'\">\s]+))?)+\s*|\s*)\/?>", '')
+    # Gerulata fix: HTMLTagRule = Rule(r"<\/?\w+((\s+\w+(\s*=\s*(?:\".*?\"|'.*?'|[\^'\">\s]+))?)+\s*|\s*)\/?>", '')
+    HTMLTagRule = Rule(r"(<([^>]+)>)", '')
```

Comment:
Prečo je tento regex lepší?

Reply:
Pretože je skoro 2x rýchlejší a robí v podstate to iste.
Keď si ho das do https://regex101.com/ a prepneš na python FLAVOR (naľavo) a do test stringu si dáš napríklad:
<h1>The Crushing Bore</h1>
Tak ten pôvodný ma 22 krokov a ten môj ma 12. (uvidíš to nad regexom v zelenom)

Reply:
Áno, ale predpokladám, že tam bol nejaký dôvod, prečo je tam takýto komplexnejší regex.

Reply:
Ok, tak napríklad toto je ten dôvod: https://rubular.com/r/tXJ5WzkE4OeIaD

Reply:
Ten pôvodný regex lepšie vyhľadáva veci, ktoré reálne vyzerajú ako HTML tagy a nie hocičo medzi dvoma zobáčikmi.

Reply:
Áno tie fix riadky odstránim. To bolo pre mňa aby som videl kde som čo menil.

Reply:
Tu by som navrhoval nechať pôvodný regex pattern, lebo je lepší (aj keď pomalší).

```suggestion
    HTMLTagRule = Rule(r"<\/?\w+((\s+\w+(\s*=\s*(?:\".*?\"|'.*?'|[\^'\">\s]+))?)+\s*|\s*)\/?>", '')
```